network/stream/stream.go

```diff
@@ -392,7 +358,7 @@ func (r *Registry) serverHandleGetRange(ctx context.Context, p *Peer, msg *GetRa
 
 	key, err := provider.ParseKey(msg.Stream.Key)
 	if err != nil {
-		return fmt.Errorf("erroring parsing stream key: %w", err)
+		return fmt.Errorf("erroring parsing stream key, stream: %s, err: %w", msg.Stream, err)
```

Comment:
Word `erroring` is not needed, as it is clear that the error is returned. I am suggesting something like `parse stream key...`. This goes to all similar errors, I know that you did not write them, but it is a nice opportunity to fix them.